README.md

```diff
@@ -1,110 +1 @@
-- Please ensure that you use a separate XkOps namespace.
-
-## 📒 Getting Started
-
-To install XkOps, please follow these steps
-
-### 🔐 Secret Manager Setup
-
-First, set up AWS secrets manager on your AWS account:
-
-- Refer to this [guide](https://docs.google.com/document/d/17fhQ0zJZtJGcWtnVD8NehUbFC-x9TrMP11XjyEFi370/edit?usp=sharing) for instructions on how to set up AWS secret manager OR
-- Use [this script](https://github.com/XgridInc/xkops/blob/master/secret-manager/secret-manager.sh) to automate the setup process.
-
-### 📥 Install XkOps
-
-1. Clone the repository and navigate to the cloned repo:
-
-    ```commandline
-    git clone https://github.com/XgridInc/xkops.git && cd xkops
-    ```
-
-2. Update values.yaml file and input your specific value for each key.
-3. Install XkOps using Helm:
-
-    ```commandline
-    helm install xkops ./helm -f values.yml
-    ```
-
-4. After successful installation, obtain the link of the XkOps frontend service to access the dashboard:
-
-    ```commandline
-    kubectl get svc -n xkops
-    ```
-
-5. Create an unclaimed volume in your cluster and delete it using the delete button on the dashboard. You can verify the volume deletion action both from the dashboard and the cluster.
-
-## 🚧 Road Map
-
-To report a new feature request or to report any issues or bugs encountered while using XkOps, please feel free to [create a new issue](https://github.com/XgridInc/xkops/issues "create a new issue") on the project's GitHub repository or contact the development team via [Slack Channel](https://join.slack.com/t/xkopscommunity/shared_invite/zt-1u8xzjvvq-B52TJ2XE861v3KDvpA9UVg). The following features are currently either in progress or planned:
-
-- [X] Checking for observability tools in your Kubernetes cluster.
-- [X] Installing tools to mitigate risk.
-- [X] Using Robusta to monitor and troubleshoot clusters.
-- [X] Employing Kubecost for cost optimization.
-- [X] Utilizing Pixie to monitor system performance.
-- [x] Deployment using Helm charts.
-- [ ] Implementation of a user interface.
-- [ ] Determining risk factors based on metrics from your Kubernetes cluster.
-- [ ] Extracting logs from pod using a logging solution
-
-## 🤝 How to contribute
-
-We invite you to contribute to XkOps, which is a community driven project.  If you plan on contributing code, kindly go through our [contribution guide](https://github.com/XgridInc/xkops/blob/master/CONTRIBUTING.md).
-
-- To report a bug or request a feature, you can submit a [GitHub issue](https://github.com/XgridInc/xkops/issues "create a new issue").
-- For real-time discussions and immediate assistance, please join our [Slack channel](https://join.slack.com/t/xkopscommunity/shared_invite/zt-1u8xzjvvq-B52TJ2XE861v3KDvpA9UVg).
-
-## 🧾 License
-
-XkOps is licensed under Apache License, Version 2.0. See [LICENSE.md](https://github.com/XgridInc/xkops/blob/master/LICENSE "LICENSE.md") for more information
+# x-workflow-2
```

Comment:
can we add readme like that of murtaza here ?

Reply:
Comments addressed.